scripts/run_eks_smoke_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+BUILD=${TRAVIS_BUILD_NUMBER:-local}
+USE_REGION=${USE_REGION:-us-east-1}
+
+handle_error() {
+    trap - EXIT
+    show_kube_info || true
+    cleanup || true
+}
+
+cleanup() {
+    delete_kube_resources
+}
+
+delete_kube_resources() {
+    kubectl delete -f $SCRIPT_DIR/eks-smoke-test/kip-nginx.yaml
+    kubectl delete -f $SCRIPT_DIR/eks-smoke-test/regular-node-nginx.yaml
+    kubectl delete -n kip-smoke-tests pod test --ignore-not-found
+    kubectl delete pod test --ignore-not-found
+}
+
+show_kube_info() {
+    kubectl get pods -A
+    kubectl get nodes
+    kubectl -n kip-smoke-tests describe pod -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0
+    kubectl logs -n kip-smoke-tests -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0r -c init-cert --tail=-1
+    kubectl logs -n kip-smoke-tests -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0 -c kip --tail=-1
+}
+
+update_vk() {
+    local version="$(git describe)"
+    local patch_init="{\"spec\":{\"template\":{\"spec\":{\"initContainers\":[{\"image\":\"elotl/init-cert:$version\",\"name\":\"init-cert\"}]}}}}"
+    local patch_kip="{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"image\":\"elotl/kip:$version\",\"name\":\"kip\"}]}}}}"
+    kubectl patch -n kip-smoke-tests statefulset kip-build-kip-provider -p "$patch_init"
+    kubectl patch -n kip-smoke-tests statefulset kip-build-kip-provider -p "$patch_kip"
+    kubectl taint node kip-build-kip-provider-0 usage=kip-smoke-tests:NoSchedule --overwrite
+}
+
+run_smoke_test_1() {
+    kubectl apply -f $SCRIPT_DIR/eks-smoke-test/regular-node-nginx.yaml
+    local curlcmd="i=0; while [ \$i -lt 300 ]; do i=\$((i+1)); curl regular-nginx.kip-smoke-tests | grep 'Welcome to nginx' && exit 0; sleep 1; done; exit 1"
+    local waitcmd="phase=\"\"; echo \"Waiting for test results from pod\"; until [[ \$phase = Succeeded ]]; do sleep 1; phase=\$(kubectl get pod -n kip-smoke-tests test -ojsonpath=\"{.status.phase}\"); if [[ \$phase = Failed ]]; then echo \$phase; kubectl get pods -A; exit 1; fi; echo \$phase; done"
```

Comment:
curlcmd and waitcmd should be defined as readonly constants out of the function. I don't see any reason to have them be local variables.